include/pfs/drp/stella/FiberTraces.h

```diff
@@ -378,28 +378,25 @@ namespace math{
    * @param yCenters: shape(nfibers * nRows)
    */
   template< typename ImageT, typename MaskT, typename VarianceT, typename T, typename U, int I >
-  bool assignITrace( FiberTraceSet< ImageT, MaskT, VarianceT > & fiberTraceSet,
+  void assignITrace( FiberTraceSet< ImageT, MaskT, VarianceT > & fiberTraceSet,
                      ndarray::Array< T, 1, I > const& traceIds,
-                     ndarray::Array< U, 1, I > const& xCenters,
-                     ndarray::Array< U, 1, I > const& yCenters );
+                     ndarray::Array< U, 1, I > const& xCenters );
```

Comment:
Need to adjust the documentation block above to reflect the removal of `yCenters`.